src/components/Profile.vue

```diff
@@ -5,29 +5,25 @@
 			<img src="../assets/mock/img/profilepic.jpeg" alt="" srcset="">
```

Comment:
I think the image url should be part of the user object that's returned from the API.